pkg/metrics/metrics.jsx

```diff
@@ -382,19 +382,25 @@ class CurrentMetrics extends React.Component {
             const merged = [];
             names.forEach((k, i) => {
                 const v = values[i];
+                let user;
                 // filter out invalid values, the empty (root) cgroup, non-services
                 if (k.endsWith('.service') && typeof v === 'number' && v != 0) {
+                    if (k.match(/^user.*user@\d+\.service.+/)) {
+                        user = true;
+                    } else {
+                        user = false;
+                    }
```

Comment:
Please prefer writing code to (1) localize variables as much as possible (so that they don't leak into a bigger scope than necessary -- easier to follow in developer's head, and easier to avoid conflicts), and (2) making them `const` whenever possible.  Also, as this is not an user name, but a boolean flag, rename it a bit:

So please write this as

    const is_user = k.match(/^user.*user@\d+\.service.+/);